src/main/java/uk/gov/ons/census/casesvc/model/entity/Case.java

```diff
@@ -114,4 +114,7 @@
   @Column(columnDefinition = "timestamp with time zone")
   @UpdateTimestamp
   private OffsetDateTime lastUpdated;
+
+  @Column(nullable = false, columnDefinition = "BOOLEAN DEFAULT false")
```

Comment:
`nullable = false` is implied

Reply:
Want me to change the rest of the columns to remove `nullable = false`